commands_test.go

```diff
@@ -1985,6 +1985,62 @@ var _ = Describe("Commands", func() {
 
 			Expect(args).To(Equal(expectedArgs))
 		})
+
+		It("should ACL LOG", func() {
```

Comment:
I think we should cause a few acl logs for the tests, so we can check we're indeed reading in the resp response. We would have caught that the current code returns an error.
Not sure if that's easy to do with the current testing set-up, but @chayim  is working on a more flexible solution at the moment.